### PR TITLE
Fixes issue #8366

### PR DESCRIPTION
**Pull request by bytetwt**
## Proposed Changes

- Fixes #issue_number
- Change 1
- Change 2
- More?

@ohcnetwork/code-reviewers

## Merge Checklist

- [x] Add specs that demonstrate bug / test a new feature.
- [x] Update [product documentation](https://docs.ohc.network).
- [x] Ensure that UI text is kept in I18n files.
- [x] Request for Peer Reviews
- [x] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [x] Completion of QA

### CODE REVIEW

File: src/Components/Common/prescription-builder/ProcedureBuilder.tsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ export default function ProcedureBuilder(props: Props<ProcedureType>) {
                   </button>
                 </div>
                 <div className="w-full">
-                  Procedure
+                  Procedure Suggestions
```
Comment (rithviknishad) [inline src/Components/Common/prescription-builder/ProcedureBuilder.tsx:67]: Let's move the text to i18n

File: .env [hunk 2]
```diff
@@ -10,3 +10,6 @@ REACT_PUBLIC_URL=https://care.ohc.network
 ESLINT_NO_DEV_ERRORS=true
 
 CARE_CDN_URL="https://egov-s3-facility-10bedicu.s3.amazonaws.com https://egov-s3-patient-data-10bedicu.s3.amazonaws.com http://localhost:4566"
+
+#.env
+CARE_API=http://127.0.0.1:9000
```
Comment (rithviknishad) [inline .env:15]: Could you discard this change? 

You can use `.env.local` file to set envs for local development as that file is git ignored.

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Comment by rithviknishad** [comment]
@Adiflickzzz Please follow contributing guidelines. https://github.com/ohcnetwork/care_fe#contributing-to-care

- [ ] Needs proper meaningful title for the PR that describes what change has been made.
- [ ] Link the issue with a closing keyword in the PR's main comment

File: .env (hunk 2 above)
Comment (bytetwt) [reply .env:15]: Done !!

File: src/Components/Common/prescription-builder/ProcedureBuilder.tsx (hunk 1 above)
Comment (bytetwt) [reply src/Components/Common/prescription-builder/ProcedureBuilder.tsx:67]: i have moved it and created a pr

**Comment by rithviknishad** [comment]
Closing as duplicate of #8443

**Pull request closed without merging**